pages/common/npm.md

```diff
@@ -4,34 +4,34 @@
 > Manage Node.js projects and their module dependencies.
 > Homepage: <https://www.npmjs.com/>.
 
-- Download and install a module globally:
+- Interactively create a package.json file:
 
-`npm install -g {{module_name}}`
+`npm init`
 
-- Download all dependencies referenced in package.json:
+- Download all the packages listed as dependencies in package.json:
 
 `npm install`
 
-- Download a given dependency required for the application to run, and add it to the package.json:
+- Download a package and add it to the list of dependencies in package.json:
 
-`npm install {{module_name}}@{{version}} --save`
+`npm install {{module_name}}`
 
-- Download a given dependency for development purposes, and add it to the package.json:
+- Download a specific version of a package and add it to the list of dependencies in package.json:
 
-`npm install {{module_name}}@{{version}} --save-dev`
+`npm install {{module_name}}@{{version}}`
 
-- Uninstall a module:
+- Download a package and add it to the list of dev dependencies in package.json:
 
-`npm uninstall {{module_name}}`
+`npm install {{module_name}} --save-dev`
 
-- List a tree of installed modules referenced in package.json:
+- Download a package and install it globally:
 
-`npm list`
+`npm install --global {{module_name}}`
 
-- List top-level globally installed modules:
+- Uninstall a package and remove it from the list of dependencies in package.json:
 
-`npm list -g --depth={{0}}`
+`npm uninstall {{module_name}}`
 
-- Interactively create a package.json file:
+- Print the dependency tree defined in package.json:
```

Comment:
This isn't technically correct, I don't think. The `list` subcommand lists the currently-installed dependency tree, and doesn't depend on the contents of `package.json`. By this I mean that you can have a package installed but not defined in package.json, and `npm list` will still show it to you.

Reply:
Good catch. I'll rephrase.

Reply:
Hmm. on second thought, I could use a suggestion here. That part of the description was in the prior version of the doc as well. I think the intent is to indicate that `npm list` only tells you about your project dependencies and not your global dependencies. 

Does anyone have a good idea on how to phrase that?

Reply:
What about this?

```suggestion
- Print a tree of locally-installed dependencies:
```

Reply:
I like it! I'll make the change and repush.